src/MiddlewareFactory.php

```diff
@@ -86,7 +86,7 @@ public function process(
 
                     /** @var mixed $response */
                     $response = (new Injector($this->container))
-                        ->invoke([$controller, $this->method], array_merge($request->getAttributes() ?? [], [$request, $handler]));
+                        ->invoke([$controller, $this->method], array_merge($request->getAttributes(), [$request, $handler]));
```

Comment:
This is throws exception when requst will be contain attribute `request` or `handler` and action will be contain arguments `$request` or `$handler`.

Reply:
Fixed.